apps/rotm/views/partials/confirmation-alert.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert-complete" role="alert">
+  <div class="alert-message">
+    <h1>{{#t}}pages.confirmation.alert-1{{/t}}
+    <div>{{#t}}pages.confirmation.alert-2{{/t}}</div></h1>
```

Comment:
What's the deal with the random div? What's it there for?

Reply:
it allows for the second part of the content to be on a new line as per the ticket

*No Div*
![screen shot 2017-11-29 at 14 00 17](https://user-images.githubusercontent.com/12494656/33378767-cc1fcd8e-d50d-11e7-88dc-e1baaf42dcc2.png)

*With Div*
![screen shot 2017-11-29 at 14 00 35](https://user-images.githubusercontent.com/12494656/33378792-e24253b6-d50d-11e7-9c0b-638cf980dea3.png)